### PR TITLE
Update rule.md

### PR DESCRIPTION
**Pull request by bahjatalaadelssw**
Shorting the URL to read easier.

### CODE REVIEW

**Comment by tiagov8** [comment]
Hi @bahjatalaadelssw 

1. Please improve the PR title... "Updated rule.md" is not descriptive enough
2. You need to follow some extra steps to shorten a URL: https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Rename-Rules. Please action
3. Add me as reviewer ;)

Cheers,

File: rules/share-the-action-items-that-came-up/rule.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
 title: After - Do you share the action items that came up after a meeting?
-uri: share-the-action-items-that-came-up
```
Comment (tiagov8) [inline rules/share-the-action-items-that-came-up/rule.md:4]: 1. Please do the other steps on https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Rename-Rules

Comment (bahjatalaadelssw) [reply rules/share-the-action-items-that-came-up/rule.md:4]: Hi @tiagov8 

All done

>1 - Please improve the PR title... "Updated rule.md" is not descriptive enough
>2 - You need to follow some extra steps to shorten a URL: https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Rename-Rules. Please action
>3 - Add me as reviewer ;)

**Review by UlyssesMaclaren — APPROVED** [review]

**Comment by tiagov8** [comment]
Hey @bahjatalaadelssw 

This has been approved by @UlyssesMaclaren , but you didn't action task 1

**Comment by bahjatalaadelssw** [comment]
Hi @tiagov8 ,

Apologies, I forgot the first message 😅

Done
>1 Please do the other steps on https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Rename-Rules

**Pull request merged**